Automattic/woocommerce-payments
# Modified the Post-merge task to provide link to QA Testing Instructions

Fixes #3607 

#### Changes proposed in this Pull Request

Modified the post-merge task in the PR template so that it is clear if the fix should be tested by QA or not.

#### Testing instructions
PR template change is useful for the release lead to check the issues in scope for QA testing at each release.

-------------------


**Post merge**
